.rubocop.yml

```diff
@@ -0,0 +1,220 @@
+AllCops:
+  RubyInterpreters:
+    - ruby
+    - rake
+    - jruby
+  Include:
+    - '**/*.rb'
+    - '**/*.gemfile'
+    - '**/*.gemspec'
+    - '**/*.rake'
+    - '**/Gemfile'
+    - '**/Rakefile'
+  Exclude:
+    - 'node_modules/**/*'
+    - 'tmp/**/*'
+    - 'vendor/**/*'
+    - '.git/**/*'
+    - 'build/**/*'
+    - 'config/**/*'
```

Comment:
Adding `.gradle`, `.idea` auto generated folders, and even extending to `.ci/.buildkite/.github` would be nice.
Or including only target folders (we add to Logstash distribution) is also okay. Just my opinion, let you decide since this doesn't break any lint feature :)